components/browser/session/src/main/java/mozilla/components/browser/session/engine/EngineObserver.kt

```diff
@@ -39,6 +40,14 @@ internal class EngineObserver(val session: Session) : EngineSession.Observer {
                 ?: "", issuer ?: "")
     }
 
+    override fun onTrackerBlocked(url: String) {
+        session.trackersBlocked += url
+    }
+
```

Comment:
This information is dependent on the page loaded and we clear that in Focus whenever we load a new page.

Reply:
OK, I will update and add this.